woocommerce/woocommerce-ios
# Edit Product: supported sale dates without a start or end date

Fixes #1704 

Ready for review after https://github.com/woocommerce/woocommerce-ios/pull/1694 is merged.

## Description
Right now we show the sale dates on the Edit Product main screen > Price row only when both sale start and end dates are specified. Let's add support for the cases without a start or end date. 

## Changes
- Fixed product remote update when there is only sale start date or sale end date
- Support product sale dates without a start or end date

## Testing 
1) On the website, take two products, one with only the start sale date, and another one with only the end sale date
2) Open the app
3) Select the product tab
4) Select a product with only the start sale date, you will be able to see the price row with only the sale start date formatted like this `Sale dates: From \(startDate)`
5) Select a product with only the end sale date, you will be able to see the price row with only the sale end date formatted like this `Sale dates: Until \(endDate)`
 

## Screenshots

| Only start date            |  Only end date | Start and End date
:-------------------------:|:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 11 Pro - 2020-01-09 at 17 49 41](https://user-images.githubusercontent.com/495617/72089588-9de6e380-330c-11ea-8781-0d957013350a.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-09 at 17 49 57](https://user-images.githubusercontent.com/495617/72089590-a0e1d400-330c-11ea-93b8-32b0e282be12.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-09 at 17 50 13](https://user-images.githubusercontent.com/495617/72089699-ccfd5500-330c-11ea-802c-552ead32f11a.png)


Update release notes:
